src/main/java/pl/zimi/zadanka/Zad6.java

```diff
@@ -1,5 +1,29 @@
 package pl.zimi.zadanka;
 
 public class Zad6 {
+    public int removeDuplicates(int[] nums) {
+        int Len = 0;
+        int numsLen = nums.length - 1;
+        int arrayRank = nums[numsLen];
 
+
+        int i = 0;
+        while (nums[arrayRank - 1] != arrayRank) {
```

Comment:
nie rozumiem tego przypuszczenia że wartość z tablicy dla index - 1 jest różne od indeksu.
Czy w zadaniu jest założenie że to muszą być kolejne liczby?
Nie widzę tam takiego założenia... muszą być posortowane z tego co widzę
[3, 10, 25] też jest posortowaną tablicą